### ISSUE TITLE
No module named `pynvml`

### ISSUE THREAD

**Issue by dylanrstewart** [rapidsai-community/rapidsai-csp-utils#63]
When running the `rapids-colab.ipynb` notebook:
```
!git clone https://github.com/rapidsai/rapidsai-csp-utils.git
!python rapidsai-csp-utils/colab/env-check.py
```

Cloning into 'rapidsai-csp-utils'...
remote: Enumerating objects: 300, done.
remote: Counting objects: 100% (129/129), done.
remote: Compressing objects: 100% (74/74), done.
remote: Total 300 (delta 74), reused 99 (delta 55), pack-reused 171
Receiving objects: 100% (300/300), 87.58 KiB | 17.52 MiB/s, done.
Resolving deltas: 100% (136/136), done.
Traceback (most recent call last):
  File "rapidsai-csp-utils/colab/env-check.py", line 1, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'

**Comment by beckernick** [comment]
cc @taureandyernv , have you run into this?

**Comment by taureandyernv** [comment]
@drstewart19 , thank you for letting us know.  At the moment, we're suggesting [everyone opt to try our Sage Maker Studio Live (SMSL)](https://rapids.ai/smsl.html) solution over Colab, as SMSL supports the latest stable and nightly versions of RAPIDS and Colab can only support 21.12.  This does require you to sign up for SMSL, but it's free.  

I'll leave the issue open, as this workaround doesn't address the issue you're facing

**Comment by lextoumbourou** [comment]
I installed `pynvml` with pip before running the `env-check.py` and everything seems to work. 

```
!pip install pynvml
```

Takes a long, long time to install though.

**Comment by taureandyernv** [comment]
We just released pip packages, and it works with Google Colab now that they've updated to Python 3.8 two days ago.  I've addressed this issue in #65
